package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "csharp",
   "publisher": "ms-vscode",
-  "version": "1.3.0",
+  "version": "1.3.0-beta1",
```

Comment:
got it! thanks

---
In reply to: [71229831](https://github.com/OmniSharp/omnisharp-vscode/pull/585#discussion_r71229831) [](ancestors = 71229831)